lldb/test/API/lldbtest.py

```diff
@@ -101,6 +101,19 @@ def execute(self, test, litConfig):
         if timeoutInfo:
             return lit.Test.TIMEOUT, output
 
+        # Temporary fix to a flaky CI error. See rdar://52221547
+        # After unit tests have finished, a race during lldb shutdown can
+        # result in this innocuous and trouble causing failure:
+        #
+        # libc++abi.dylib: terminating with uncaught exception of type \
+        # std::__1::system_error: recursive_mutex lock failed: Invalid argument
+        if (
+            'RESULT: PASSED' in err and
+            'recursive_mutex lock failed' in err and
+            exitCode != 0
+        ):
+            return lit.Test.PASS, output
```

Comment:
I think `FLAKYPASS` would be a better fit.

Reply:
done